packit_service/utils.py

```diff
@@ -226,7 +226,7 @@ def get_pr_comment_parser(prog: str, description: str, epilog: str) -> argparse.
         epilog=epilog,
```

Comment:
Sorry for unrelated comment, typo in the commit message of ca53b4bd870eaec0083c3d6769d3b828bb87e759

formatiing -> formatting

thanks <3